modules/rostests/apitests/shell32/SHGetKnownFolderPath.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Test for SHGetKnownFolderPath
+ * COPYRIGHT:   Copyright 2025 Petru Răzvan (petrurazvan@proton.me)
+ */
+#include "shelltest.h"
+
+typedef HRESULT (WINAPI *PSHGETKNOWNFOLDERPATH)(REFKNOWNFOLDERID, DWORD, HANDLE, PWSTR *);
+
+static PSHGETKNOWNFOLDERPATH pSHGetKnownFolderPath = NULL;
+
+START_TEST(SHGetKnownFolderPath)
+{
+    HRESULT hr;
+    HINSTANCE hShell32;
+    PWSTR path;
+    HRESULT result;
+
+    hShell32 = LoadLibraryW(L"shell32.dll");
+    if (!hShell32)
+    {
+        skip("Failed to load shell32.dll\n");
+        return;
+    }
```

Comment:
The shell32_apitest already links against shell32, so don't use the LoadLibrary but instead go with `GetModuleHandleW(L"shell32.dll");`, and the FreeLibrary wouldn't be necessary.
```suggestion
    hShell32 = GetModuleHandleW(L"shell32.dll");
    if (!hShell32)
    {
        skip("Failed to find shell32.dll\n");
        return;
    }
```